metadata.json

```diff
@@ -58,6 +58,6 @@
     }
   ],
   "pdk-version": "2.1.0",
-  "template-url": "https://github.com/puppetlabs/pdk-templates#2.1.0",
-  "template-ref": "tags/2.1.0-0-ga675ea5"
+  "template-url": "https://github.com/puppetlabs/pdk-templates#main",
+  "template-ref": "heads/main-0-g03daa92"
```

Comment:
Are we using features from `main`? If not, let's put this back to 2.1.0.

Reply:
:+1: